### PR TITLE
Advanced Qgis updated

### PR DESCRIPTION
**Pull request by VignaPurohit**

### CODE REVIEW

File: advanced-qgis.Rmd [hunk 1]
```diff
@@ -283,7 +283,7 @@ knitr::include_graphics('images/advanced_qgis/batch01.png')
 knitr::include_graphics('images/advanced_qgis/batch02.png')
 ```
 
-3. Now that we have centered on our area of interest, under the **Processing** menu and select **toolbox**. Now in *Processing Toolbox*, search for **clip**, under **Vector overlay &rarr; Clip** right-click and select **Execute as Batch Process..**. 
+3. Now that we have centered on our area of interest,Select the ``places`` layer. Go to the the **Processing** menu and select **toolbox**. Now in *Processing Toolbox*, search for **clip**, under **Vector overlay &rarr; Clip** right-click and select **Execute as Batch Process..**.
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:286]: 'select' should be small case

File: advanced-qgis.Rmd [hunk 2]
```diff
@@ -382,7 +382,7 @@ We will now learn how to build a model that runs the above processing steps in a
 knitr::include_graphics('images/advanced_qgis/modeler1.png')
 ```
 
-2. Launch the modeler from **Processing &rarr; Graphical Modeler** 
+2. Launch the modeler from **Processing &rarr; Model Designer**
```
Comment (spatialthoughts) [inline advanced-qgis.Rmd:385]: Good catch. Let's fix all other instances of 'graphical modeler' to 'model designer' for this course in this same PR. And send a separate PR for qgis-tutorials.

**Review by spatialthoughts — COMMENTED** [review]
Some comments below. As noted, also update the screenshot 10 of section 4.3.4

File: advanced-qgis.Rmd (hunk 2 above)
Comment (VignaPurohit) [reply advanced-qgis.Rmd:385]: Updated.
